keycloak/keycloak-admin-ui
# Fix/edit user label

## Motivation
Fix for #1627

## Verification Steps
1. Go to users menu item, credentials tab and set up a new credential.
2. User label should be empty by default, select a pencil icon and enter a text in the text box and select a tick icon.
3. Observe the user label populated with the correct text on the first try. Edit a few times to double-check.

## Checklist:
